views/event/create.ejs

```diff
@@ -3,27 +3,54 @@
 <head>
     <% include ../template/head.ejs %>
     <script>
+        // default date
+        window.onload= function() {
+            let now = new Date();
+            let day = ("0" + now.getDate()).slice(-2);
+            let month = ("0" + (now.getMonth() + 1)).slice(-2);
+            let today = now.getFullYear()+"-"+(month)+"-"+(day) ;
+            $("#time").val(today);
+        }
+        // validate URL
+        function validURL(str) {
+            var pattern = new RegExp('^(https?:\\/\\/)?'+ // protocol
+                '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.)+[a-z]{2,}|'+ // domain name
+                '((\\d{1,3}\\.){3}\\d{1,3}))'+ // OR ip (v4) address
+                '(\\:\\d+)?(\\/[-a-z\\d%_.~+]*)*'+ // port and path
+                '(\\?[;&a-z\\d%_.~+=-]*)?'+ // query string
+                '(\\#[-a-z\\d_]*)?$','i'); // fragment locator
+            return !!pattern.test(str);
+        }
     	$(function() {
             $("#submit-button").click(function() {
                 $("#submit-button").attr('disabled', true);
                 var validInput = true;
                 $("#error-msg").html("");
-                if (($("#title").val() == '') | ($("#time").val() == '') | ($("#location").val() == '')) {
+                if (($("#title").val() == '') | ($("#time").val() == '') | ($("#location").val() == '') |
+                    ($("#link").val() == '') | ($("#type").val() == '') | ($("#description").val() == '') | ($("#image").val() == '')) {
```

Comment:
Empty image or link should be acceptable (if so there should be a default image as already impelemented)